<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{
    padding: -1;
} 
QPushButton::menu-indicator{image:None;}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="minimumSize">
         <size>
          <width>540</width>
          <height>120</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>120</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="Home">
         <attribute name="title">
          <string> 主页</string>
         </attribute>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>30</y>
            <width>48</width>
            <height>48</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res.qrc">
            <normaloff>:/Shape/Shape/text.png</normaloff>:/Shape/Shape/text.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>281</width>
            <height>81</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="DrawPt">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/Shape/Shape/point.png</normaloff>:/Shape/Shape/point.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DrawLine">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/Shape/Shape/line.png</normaloff>:/Shape/Shape/line.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>48</width>
               <height>48</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DrawEllipse">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/Shape/Shape/ellipse.png</normaloff>:/Shape/Shape/ellipse.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>48</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DrawRect">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/Shape/Shape/rectangle.png</normaloff>:/Shape/Shape/rectangle.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>57</width>
               <height>48</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="Insert">
         <attribute name="title">
          <string>插入</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="title">
          <string>Page</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabView">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>400</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="start">
         <attribute name="title">
          <string> 开始</string>
         </attribute>
         <widget class="QPushButton" name="pushButton_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>211</width>
            <height>181</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>280</y>
            <width>191</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>20</y>
            <width>181</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;最近使用的文件&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>20</y>
            <width>151</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;开始绘制&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>580</x>
            <y>20</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;当前时间&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>72</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>  文件名</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>150</y>
            <width>81</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string> 打开时间</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="draw_1">
         <attribute name="title">
          <string> 画面1</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="MyView" name="m_view"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_Close">
     <property name="title">
      <string>关闭</string>
     </property>
     <addaction name="action_Current"/>
     <addaction name="action_All"/>
    </widget>
    <widget class="QMenu" name="menu_SaveAs">
     <property name="title">
      <string>另存为</string>
     </property>
     <addaction name="action_Pic"/>
     <addaction name="action_PDF"/>
     <addaction name="actionSVG"/>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="menu_SaveAs"/>
    <addaction name="separator"/>
    <addaction name="action_Publish"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="menu_Close"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act1">
   <property name="text">
    <string>圆点</string>
   </property>
  </action>
  <action name="act2">
   <property name="text">
    <string>×形点</string>
   </property>
  </action>
  <action name="act3">
   <property name="text">
    <string>坐标画点</string>
   </property>
  </action>
  <action name="actLine_1">
   <property name="text">
    <string>直接画线</string>
   </property>
  </action>
  <action name="actLine_2">
   <property name="text">
    <string>坐标画线</string>
   </property>
  </action>
  <action name="actLine_3">
   <property name="text">
    <string>角度和长度画线</string>
   </property>
  </action>
  <action name="actRect_1">
   <property name="text">
    <string>直接画矩形</string>
   </property>
  </action>
  <action name="actRect_2">
   <property name="text">
    <string>坐标画矩形</string>
   </property>
  </action>
  <action name="actEllipse_1">
   <property name="text">
    <string>直接画圆</string>
   </property>
  </action>
  <action name="actEllipse_2">
   <property name="text">
    <string>坐标画圆</string>
   </property>
  </action>
  <action name="actRect_3">
   <property name="text">
    <string>画圆角矩形</string>
   </property>
  </action>
  <action name="NewView">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="Open">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="Save">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="Print">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/print.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="SaveAs">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/save_as.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="Previous">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/previous.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>向前</string>
   </property>
  </action>
  <action name="Next">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/next.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>向后</string>
   </property>
  </action>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset>
     <normalon>:/Icon/Icon/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_Publish">
   <property name="text">
    <string>发布</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>打印</string>
   </property>
  </action>
  <action name="action_Current">
   <property name="text">
    <string>当前图形</string>
   </property>
  </action>
  <action name="action_All">
   <property name="text">
    <string>所有图形</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Pic">
   <property name="text">
    <string>图片</string>
   </property>
  </action>
  <action name="action_PDF">
   <property name="text">
    <string>PDF文件</string>
   </property>
  </action>
  <action name="actionSVG">
   <property name="text">
    <string>SVG</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header location="global">myview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
